.github/workflows/main.yml

```diff
@@ -63,8 +63,8 @@ jobs:
       - name: Run tests
         run: npm run test
 
-  cypress_tests:
-    name: Cypress tests
+  UI_tests:
```

Comment:
Thought Cypress tests doesn't make sense anymore, does UI tests make sense? To distinguish from e2e?